### DISCUSSION TITLE
RSS feed for the blog?

### DISCUSSION THREAD

**Discussion by jschuur** [zed-industries/zed#6621] [Feature Requests]
Could you add an RSS feed for https://zed.dev/blog please?

Ideally with auto-discovery (`<link rel="alternate" type="application/rss+xml" title="Zed Blog" href="/rss.xml" />`).

**Comment by ulzu** [comment]
yes, please

**Comment by JosephTLyons** [comment]
We have a link for an RSS feed, we just aren't exposing it at all on the website:

https://zed.dev/blog.rss
